docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -690,7 +690,7 @@ Status Codes:
 
 ### Resize a container TTY
 
-`POST /containers/(id)/resize?h=<height>&w=<width>`
+`POST /containers/(id)/resize`
```

Comment:
Probably, based on the tests here in [docker_api_reize_test.go](https://github.com/docker/docker/blob/e98a8815704c4c4007d45a88f88ab6a45959066e/integration-cli/docker_api_resize_test.go)

Perhaps I should have kept the original format here for /container  (ie. w `h=<height>&w=<width>`) and modify "Exec Resize" instead?
https://docs.docker.com/reference/api/docker_remote_api_v1.19/#exec-resize